nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/CommentUtils.java

```diff
@@ -131,15 +131,26 @@ public static void checkReceivedMail(List<MailMessage> mails, DocumentModel comm
 
     public static String getExpectedMailContent(DocumentModel commentDocModel, DocumentModel commentedDocModel,
             Event event, String commentEventType) {
-        URL url = CommentUtils.class.getResource("/templates/commentNotificationMail.txt");
         try {
-            String content = Files.readString(Paths.get(url.toURI()));
+            String template = "commentNotificationMail.txt";
+            Serializable parentAuthor = "";
+            Serializable parentText = "";
+            DocumentModel parentComment = (DocumentModel) event.getContext().getProperties().get(PARENT_COMMENT);
+            if (COMMENT_DOC_TYPE.equals(parentComment.getType())) {
```

Comment:
as discussed on F2F, we should checks the annotation type too here and in the `quotedComments.ftl` is why i proposed too to remove the whole `quotedComments.ftl` and integrate the whole controle in the base one and only check if `parentAuthor` / `parentText` is not empty and then add the its value. in this last point its mainly a proposal and others feedback are welcome

Reply:
The ftl is done and isolates the part of the quoted comments. That's what templating is handy for.
In this case I could spot the ftl check quickly thanks to its isolation rather than looking it up in baseComment.ftl.
The job is already done (the comment part is already in its own template). So it costs nothing to keep it as is.

Reply:
quotedComments.ftl has been dropped